cli/cli_app.go

```diff
@@ -148,7 +149,7 @@ AUTHOR(S):
 `
 
 // This uses the constraint syntax from https://github.com/hashicorp/go-version
-const defaultTerraformVersionConstaint = ">= v0.12.0"
+const defaultTerraformVersionConstaint = ">= v0.13.0"
```

Comment:
Why do you remove this now? If we have to make a change to 0.12, we'll have to maintain two branches.

Reply:
You're right. It still works with 0.12. I put it back and run the tests in both 0.12 and 0.13 now